arm-sql/2014-04-01/swagger/replicationLinks.json

```diff
@@ -0,0 +1,394 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database API spec",
+    "description": "The Azure SQL Database management API provides a RESTful set of web services that interact with Azure SQL Database services to manage your databases. The API enables users to create, retrieve, update, and delete databases.",
```

Comment:
Title definitely.  

For description, I'm not sure, when I jumped on this project all the descriptions were in place and were the same.  @amarzavery any opinion?

---
In reply to: [85586080](https://github.com/Azure/azure-rest-api-specs/pull/643#discussion_r85586080) [](ancestors = 85586080)